java/java.source.base/src/org/netbeans/modules/java/source/NoJavacHelper.java

```diff
@@ -66,7 +66,7 @@ public void run() {
                     theUnsafe.setAccessible(true);
                     Unsafe unsafe = (Unsafe) theUnsafe.get(null);
                     Class scopeClass = Class.forName("com.sun.tools.javac.code.Scope");
-                    unsafe.defineClass("com.sun.tools.javac.code.Scope$WriteableScope", classData, 0, classData.length, scopeClass.getClassLoader(), scopeClass.getProtectionDomain());
+                    //unsafe.defineClass("com.sun.tools.javac.code.Scope$WriteableScope", classData, 0, classData.length, scopeClass.getClassLoader(), scopeClass.getProtectionDomain());
```

Comment:
I'd suggest to change this to a reflection call - this improves the behavior when running on JDK 8 without nb-javac (prevents exceptions).

Reply:
had used reflection for the method call in recent commit